Packages/vcs/Lib/animate_helper.py

```diff
@@ -639,7 +653,7 @@ def run(self):
 
       # this is how you allow the GUI to process events during
       # animation creation
-      time.sleep(0.001)
+      #time.sleep(0.001)
```

Comment:
I'm not positive it needs to be removed yet. @dakoop added this for the GUI otherwise it would freak out. But the GUI is broken right now so I can't test. Hence the "comment out" rather than delete.